script.js

```diff
@@ -2,11 +2,98 @@
 function setup() {
   const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
+
 }
 
 function makePageForEpisodes(episodeList) {
   const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+  const mainEle = document.createElement('div');
+  mainEle.id = 'main'
+  rootElem.appendChild(mainEle)
+
+  //creating eposides and display them with their names and summary.
+  for (let i = 0 ; i < episodeList.length ; i++ ){
```

Comment:
it will be better if you use forEach ,instead of the normal if loo, to navigate array of episodes objects